src/wr_hsi_sensing/README.md

```diff
@@ -1,12 +1,41 @@
 # wr_hsi_sensing
 
 @defgroup wr_hsi_sensing wr_hsi_sensing
-@brief Provides GPS and IMU data to the ROS network
+@brief Provides GPS data to the ROS network
 
-This package contains nodes for the GPS sensor and IMU sensor on the WReaper board.  These sensors communicate directly with the hardware over the I2C bus.
+This package contains nodes for the ArduSimple RTK GNSS board, a mapping node that maps the ArduSimple /gps/fix topic to a message used in our ROS environment, and a node for mapviz, a visualization tool.
 
-The current IMU implementation doesn't actually use the IMU (due to a thermal failure at competition), but rather uses the movement of the GPS over time to derive a really coarse estimate of the heading of the rover.
+There is also an old node "previous_gps_node.py" that was used when we were using an old GPS sensor to calculate IMU heading data as a temporary fix during a competition.
+It is no longer used.
+
+# Launching Nodes
+
+## Build
+- Initialize the project: `./assemble.py init`
+- Source the workspace: `source setup.sh`
+- Clean (can help if errors occur): `./assemble.py clean`
+- Build the project: `./assemble.py build` (install dependences)
```

Comment:
It's good that you guys included this as part of the documentation, but it is unnecessary as it's already covered in [this page](https://wisconsinrobotics.github.io/WRover_Software/group__wr__system__setup__dev.html). For this README, focus on GPS-specific setup steps.

Reply:
Resolved in 861187c65573128bc2c3438a02f8fc8716f9f2c1